rpi_backlight/cli.py

```diff
@@ -120,6 +124,18 @@ def main():
                     backlight.power = True
                 with backlight.fade(duration=args.duration):
                     backlight.brightness = 100
+        elif args.set_power == "on":
+            if backlight.power:
+                return
+            backlight.power = True
+            if args.duration:
+                with backlight.fade(duration=args.duration):
+                    backlight.brightness = 100
+        elif args.set_power == "off":
+            if args.duration:
+                with backlight.fade(duration=args.duration):
+                    backlight.brightness = 0
+            backlight.power = False
         else:
             backlight.power = True if args.set_power == "on" else False
```

Comment:
This branch is now dead code

Reply:
Ive moved the new path stuff to
https://github.com/j-coopz/rpi-backlight
So this branch can still be about the previous issues

Reply:
No idea what that means, but please update if you want to get this PR merged

Reply:
Hi Linus, could you please elaborate what you want updated in this code?